README.md

```diff
@@ -1,424 +1,159 @@
-
-### Cloud Text-to-Speech
-
-- `gem install google-cloud-text_to_speech`
-- [google-cloud-text_to_speech README](google-cloud-text_to_speech/README.md)
-- [google-cloud-text_to_speech API documentation](https://googleapis.dev/ruby/google-cloud-text_to_speech/latest)
-- [google-cloud-text_to_speech on RubyGems](https://rubygems.org/gems/google-cloud-text_to_speech/)
-- [Google Cloud Text-to-Speech documentation](https://cloud.google.com/text-to-speech/docs)
-
-### Cloud Trace
-
-- `gem install google-cloud-trace`
-- [google-cloud-trace README](google-cloud-trace/README.md)
-- [google-cloud-trace instrumentation documentation](./google-cloud-trace/INSTRUMENTATION.md)
-- [google-cloud-trace on RubyGems](https://rubygems.org/gems/google-cloud-trace)
-- [Google Cloud Trace documentation](https://cloud.google.com/trace/docs/)
-
-### Transcoder API
-
-- `gem install google-cloud-video-transcoder`
-- [google-cloud-video-transcoder README](google-cloud-video-transcoder/README.md)
-- [google-cloud-video-transcoder on RubyGems](https://rubygems.org/gems/google-cloud-video-transcoder)
-- [Google Cloud Transcoder API documentation](https://cloud.google.com/transcoder/docs/)
-
-### Cloud Translation API
-
-- `gem install google-cloud-translate`
-- [google-cloud-translate README](google-cloud-translate/README.md)
-- [google-cloud-translate API documentation](https://googleapis.dev/ruby/google-cloud-translate/latest)
-- [google-cloud-translate on RubyGems](https://rubygems.org/gems/google-cloud-translate)
-- [Google Cloud Translation API documentation](https://cloud.google.com/translation/docs)
-
-### Video Intelligence API
+### Authentication
 
-- `gem install google-cloud-video_intelligence`
-- [google-cloud-video_intelligence README](google-cloud-video_intelligence/README.md)
-- [google-cloud-video_intelligence API documentation](https://googleapis.dev/ruby/google-cloud-video_intelligence/latest)
-- [google-cloud-video_intelligence on RubyGems](https://rubygems.org/gems/google-cloud-video_intelligence)
-- [Google Cloud Video Intelligence API documentation](https://cloud.google.com/video-intelligence/docs)
+Most API calls must be accompanied by authentication information proving that
+the caller has sufficient permissions to make the call. For an overview of
+authentication with Google, see https://cloud.google.com/docs/authentication.
 
-### Cloud Vision API
+These API client libraries provide several mechanisms for attaching credentials
+to API calls.
 
-- `gem install google-cloud-vision`
-- [google-cloud-vision README](google-cloud-vision/README.md)
-- [google-cloud-vision API documentation](https://googleapis.dev/ruby/google-cloud-vision/latest)
-- [google-cloud-vision on RubyGems](https://rubygems.org/gems/google-cloud-vision)
-- [Google Cloud Vision API documentation](https://cloud.google.com/vision/docs)
+*   If your application runs on an Google Cloud Platform hosting environment
```

Comment:
I feel it would be worth mentioning authentication using `gcloud` as well which comes handy during developement

Reply:
That's a good question. I'm not sure how much we're encouraging people to do this, because while it can be handy, there are also issues with it. I'll try to find out what our current messaging is.